include/revng/Pipeline/Step.h

```diff
@@ -0,0 +1,387 @@
+#pragma once
```

Comment:
So, `Pipe`s have no common base class.
I'd like to see a `Pipe.h` file with maybe some `concept` and some doc.
Maybe we could also move some `PipeWrapper*` classes that are now defined here into `Pipe.h`.